app/swagger/readme.md

```diff
@@ -1,5 +1,6 @@
 # Display the Swagger Doc
 - The doc is served from the endpoint: `GET` `v0/apidocs`
+- [Swaggeer docs](https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/) are published or can be generated locally.
```

Comment:
Neato. This belongs in it's own section at the top of the readme IMO. Maybe like:

```
# View Swagger Deployed to DEV
[We host the SwaggerUI here](https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/), which points itself at our `http://dev-api.va.gov/v0/apidocs` endpoint.

# Display the Swagger Doc Yourself
...blah...
...blah...
```